test/functional/tests/cli/test_manual_casadm_flush.py

```diff
@@ -0,0 +1,86 @@
+#
+# Copyright(c) 2022 Intel Corporation
+# SPDX-License-Identifier: BSD-3-Clause
+#
+
+import pytest
+import time
+
+from core.test_run_utils import TestRun
+from test_utils.size import Size, Unit
+from core.test_run import TestRun
+from storage_devices.disk import DiskType, DiskTypeSet, DiskTypeLowerThan
+from test_tools.fio.fio import Fio
+from test_tools.fio.fio_param import ReadWrite, IoEngine
+from api.cas import casadm
+from api.cas.cache_config import CacheMode, CleaningPolicy
+from test_utils.os_utils import Udev
+
+
+@pytest.mark.CI
+@pytest.mark.require_disk("cache", DiskTypeSet([DiskType.optane, DiskType.nand]))
+@pytest.mark.require_disk("core", DiskTypeLowerThan("cache"))
+def test_cleaning_policy():
+    """
+    Title: test_manual_casadm_flush
+    description: | The test is to see if dirty data will be removed from the Cache
+    after changing the cleaning policy from NOP to one that expects a flush.
+    pass_criteria:
+        - Cache is successfully populated with dirty data
+        - Execution of each casadm command clears the given core/cache
+        - There is no dirty data after the policy change
+    """
+    with TestRun.step("Prepare devices."):
+        cache_disk = TestRun.disks["cache"]
+        cache_disk.create_partitions([Size(1, Unit.GibiByte)])
+        cache_dev = cache_disk.partitions[0]
+
+        core_disk = TestRun.disks["core"]
+        core_disk.create_partitions([Size(1, Unit.GibiByte)])
+        core_dev = core_disk.partitions[0]
+
+        cache = casadm.start_cache(cache_dev, cache_mode=CacheMode.WB)
+        core = cache.add_core(core_dev)
+        cache.set_cleaning_policy(CleaningPolicy.nop)
+        dev_id = 1
+        Udev.disable()
+
+    with TestRun.step("Populate cache with dirty data."):
+        write_dirty_data(1, core, dev_id)
+        if cache.get_dirty_blocks() <= Size.zero():
+            TestRun.fail("Failed to populate Cache with dirty data")
+
+    with TestRun.step(
+        "Perform casadm -E command and check whether dirty data is cleared from core"
+    ):
+        TestRun.executor.run_expect_success(f"casadm -E -i {dev_id} -j {dev_id}")
```

Comment:
while writing tests try to provide command without shortcuts as example instead of `-E` use `--flush-core`